specs/commandline/verify.md

```diff
@@ -41,7 +55,7 @@ An example of `trustpolicy.json`:
         {
```

Comment:
Could you change the link of [Notation Directory Structure](https://github.com/notaryproject/notation/blob/main/specs/directory.md) to https://notaryproject.dev/docs/tutorials/directory-structure/? I will create a PR to remove the out-of-dated docs in the Notation repo later.